clients/pkg/promtail/targets/docker/targetmanager_test.go

```diff
@@ -89,7 +89,7 @@ func Test_TargetManager(t *testing.T) {
 	require.True(t, ta.Ready())
 
 	require.Eventually(t, func() bool {
-		return len(entryHandler.Received()) >= 5
+		return len(entryHandler.Received()) >= 6
```

Comment:
what caused this value to change? I don't see any changes to the test setup.

Reply:
I inserted a Very Long Log line into `flog.log` to test against (over 65k). I had to increase by one here to read in that extra line.

Reply:
got it, and that's a binary file which is why I'm not seeing the diff, thanks!